### ISSUE TITLE
How to enable debug?

### ISSUE THREAD

**Issue by ghost** [tkf/emacs-request#23]
I've tried setting `resquest-log-level` to 'debug and `verbose...

**Comment by ardumont** [comment]
Hello,

I was trying to do that myself and i think i found out.

Here are the custom defined:
```lisp
(defcustom request-log-level -1
  "Logging level for request.
One of `error'/`warn'/`info'/`verbose'/`debug'.
-1 means no logging."
  :group 'request)

(defcustom request-message-level 'warn
  "Logging level for request.
See `request-log-level'."
  :group 'request)
```

Here are the detailed possible levels:
```lisp
(defconst request--log-level-def
  '(;; debugging
    (blather . 60) (trace . 50) (debug . 40)
    ;; information
    (verbose . 30) (info . 20)
    ;; errors
    (warn . 10) (error . 0))
  "Named logging levels.")
```

I do not get completely why there are 2 customs but for request to be ultra verbose this should do:
```lisp
(custom-set-variables '(request-log-level 'blather)
                                   '(request-message-level 'blather))
```

Also, if you use ielm, you could just wrap your code inside a let.

Example using org-trello (which uses request inside):
```lisp
ELISP> (let ((org-trello--user-logged-in "ardumont")
             (orgtrello-log-level orgtrello-log-trace)
             (request-log-level 'blather)
             (request-message-level 'blather))
(orgtrello-controller-load-keys)
(orgtrello-query-http-trello (orgtrello-api-get-me) 'sync))
```

Of course, change 'blather to whatever you wish for.

Cheers,

**Comment by titaniumbones** [comment]
if you would like to subit a PR with improved docs please do so!! Thank you.

**Comment by titaniumbones** [comment]
chosing this as fixed in 8a6c15763b300b46c5179bd02e26af28e83288d3.
